examples/context-propagation/rxjava/build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 
 dependencies {
     implementation project(':core')
-    implementation project(':rxjava')
+    implementation project(':rxjava3')
```

Comment:
How about changing the classpath of `rxjava3` from `com.linecorp.armeria.*.rxjava` to `com.linecorp.armeria.*.rxjava3`
Because the classpath of rxjava2 is `com.linecorp.armeria.*.rxjava2` and they do not share any code.

Reply:
Ah thanks, I will do the same thing for other modules.